agent/api/task_unix.go

```diff
@@ -22,3 +22,8 @@ const (
 func (task *Task) adjustForPlatform() {}
 
 func getCanonicalPath(path string) string { return path }
+
+func (task *Task) CgroupEnabled() bool {
```

Comment:
`CgroupEnabled` will perform sanity checks and check whether the cgroup code paths needs to be activated. We only have a stub here and @petderek probably meant that he's changed this method and implemented the actual functionality in his branch.